docs/src/tutorials/exporting.md

```diff
@@ -1,51 +1,173 @@
+it's generally not necessary to download data as it can be rebuilt from the primary data store.
+
+For services designed to hold persistent data, see each service's page for instructions:
+
+- [MySQL](../add-services/mysql/_index.md#exporting-data)
+- [PostgreSQL](../add-services/postgresql.md#exporting-data)
+- [MongoDB](../add-services/mongodb.md#exporting-data)
+- [InfluxDB](../add-services/influxdb.md#exporting-data)
+
+## 4. Get the environment variables
+
+Environment variables can contain critical information such as tokens or additional configuration options for your app.
+
+Environment variables beginning with:
 
-## Get the environment variables
+- `env:` are exposed [as Unix environment variables](../development/variables/_index.md#top-level-environment-variables).
+- `php:` are interpreted [as `php.ini` directives](../development/variables/_index.md#php-specific-variables).
 
-If your project uses some environment variable (tokens, ...) it can be helpful to backup them if you didn't store them separately.
+All other variables are [part of `$PLATFORM_VARIABLES`](../development/variables/use-variables.md#use-platformsh-provided-variables).
 
-As stated in the Console, several possibilities exist for the environment variables.
+To back up your environment variables:
 
-* Variables beginning with `env:` are exposed as Unix environment variables
-* Variables beginning with `php:` are interpreted as `php.ini` directives.
-* All other variables are part of the environment `PLATFORM_VARIABLES` variable
+{{< codetabs >}}
+
+---
+title=Using the CLI
+file=none
+highlight=false
+---
+
+1. Get the variable's values with:
+
+   ```bash
+   platform ssh --project {{< variable "PROJECT_ID" >}} --environment {{< variable "ENVIRONMENT" >}} -- 'echo $PLATFORM_VARIABLES | base64 -d | jq'
+   ```
+
+   Note that you can also get all the environment variable values with:
+
+   ```bash
+   platform ssh --project {{< variable "PROJECT_ID" >}} --environment {{< variable "ENVIRONMENT" >}} -- env
+   ```
+
+2. Store the data on your local computer, for example in an encrypted text file for sensitive tokens.
+
+<--->
+
+---
+title=In the Console
+file=none
+highlight=false
+---
 
-More details can be found [on the environment page](https://docs.platform.sh/administration/web/configure-environment.html#variables)
+1. In the [Console](https://console.platform.sh/), open your project and click **{{< icon settings >}}**.
+2. Click **Project Settings {{< icon chevron >}}**.
+3. Click **Variables** and access your variable's values and settings.
+4. Store the data on your local computer, for example in an encrypted text file for sensitive tokens.
 
-You can access the content of the environment variable through the Console unless the `--sensitive true` flag was set.
+Note that in the Console, you can't access the value of variables that have been created using the [`--sensitive true` flag](../development/variables/set-variables.md#variable-options).
```

Comment:
:hammer_and_wrench: Given that you're talking about in the Console, it's not ideal to describe it in terms of CLI flags.
```suggestion
Note that in the Console, you can't access the value of variables that have [marked as sensitive](../development/variables/set-variables.md#variable-options).
```